src/main/java/org/commcare/cases/util/StringUtils.java

```diff
@@ -134,4 +135,18 @@ private static int LevenshteinDistance(String s0, String s1) {
         // the distance is the cost for transforming all letters in both strings
         return cost[len0 - 1];
     }
+
+    /**
+     * Converts a string to a string array.
+     */
+    public static String[] toArray(String str) {
+
+        String[] myArray = new String[str.length()];
+
+        for (int i = 0; i < str.length(); i++) {
+            myArray[i] = str.substring(i, i + 1);
```

Comment:
This might be more succinct using the String `charAt()` function (and possibly this whole function should return a `char[]` rather than a `String[]`)